NuGetPackageVerifier.json

```diff
@@ -1,4 +1,18 @@
 {
+    "adx": {
+        "rules": [
+            "DefaultCompositeRule"
+        ],
+        "packages": {
+            "Microsoft.VisualStudio.LanguageServices.Razor": {
+                "Exclusions": {
+                    "WRONG_JSONNET_VERSION": {
+                        "Microsoft.VisualStudio.LanguageServices.Razor; .NETFramework,Version=v4.6": "This package intentionally depends on version 8.0.3 of Newtonsoft.Json because of the dependency on Roslyn."
+                    }
```

Comment:
This is not true. The dependency is on Visual Studio and it's shipped version of visual studio.